tests/functional/object/mcg/test_s3_vector.py

```
@@ -233,3 +234,374 @@ def test_s3_vector_operations(self, vector_bucket_factory, mcg_obj):
+                bucket_info.get("ResponseMetadata", {}).get("HTTPStatusCode") == 200
+            ), f"Get operation failed for {random_bucket}: {bucket_info}"
+            logger.info(f"✓ Get operation successful for {random_bucket}")
+
+            # Step 6: Create duplicate bucket with same name
+            logger.info("Attempting to create duplicate bucket")
+            duplicate_name = valid_bucket_names[0]
+            try:
+                vector_utils.create_vector_bucket(s3vectors_client, duplicate_name)
+                assert (
+                    False
+                ), f"Duplicate bucket creation should have failed for {duplicate_name}"
```

Ruff (0.15.15)

[warning] Assertion always fails, replace with `pytest.fail()`

(PT015)